core/spec/services/spree/import_service/products/create_spec.rb

```
@@ -0,0 +1,80 @@
+          .and(change(Spree::ProductProperty, :count).by(2))
+      end
+
+      it 'assigns valid attributes to new resources' do
+        service.call
+
+        variant = Spree::Variant.find_by(sku: row[:sku])
+
+        expect(variant.attributes).to match(a_hash_including(expected_variant_attributes))
+        expect(variant.product.attributes).to match(a_hash_including(expected_product_attributes))
+      end
+    end
```

RuboCop (1.75.5)

[convention] Example group has too many memoized helpers [7/5]

(RSpec/MultipleMemoizedHelpers)